src/index.html

```diff
@@ -11,8 +11,56 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wdth,wght@0,75..100,100..900;1,75..100,100..900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__Photo"
+        src="images/imac.jpeg"
+        alt="iMac"
+      />
+      <h2 class="card__Name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <div class="card__index">Product code: 195434</div>
```

Comment:
You need to use the semantic tag for text, fix it everywhere
```suggestion
      <p class="card__index">Product code: 195434</p>
```